pom.xml

```diff
@@ -285,28 +285,28 @@
 										</goals>
 									</pluginExecutionFilter>
 									<action>
-										<ignore></ignore>
-									</action>
-								</pluginExecution>
-								<pluginExecution>
-									<pluginExecutionFilter>
-										<groupId>
-											com.nativelibs4java
-										</groupId>
-										<artifactId>
-											maven-jnaerator-plugin
-										</artifactId>
-										<versionRange>
-											[0.12,)
-										</versionRange>
-										<goals>
-											<goal>generate</goal>
-										</goals>
-									</pluginExecutionFilter>
-									<action>
-										<ignore></ignore>
+<!--										<ignore></ignore>-->
```

Comment:
... and this one is illegal too, that's why I commented them out, in addition to commenting out the JNAerator generation targets.